### DISCUSSION TITLE
Problem with pybricks.parameters import Axis

### DISCUSSION THREAD

**Discussion by ZPhilo** [pybricks/support#1051] [Q&A]
When I try the "Reading acceleration and angular velocity on one axis" example from documentation, Pybricks code replies that 
"ImportError: can't import name Axis"
Do I do something wrong?

**Comment by laurensvalk** [comment]
> Do I do something wrong?

That's hard to say :smile: Could you share your code example?

This feature was added only recently. Did you install the latest firmware on your hub? If you click on the hub at the bottom right, it should say something like this:

![image](https://user-images.githubusercontent.com/12326241/234865766-bfc5322e-9c3a-4c03-bd5f-d4f7a749525b.png)

**Reply by ZPhilo** [reply]
Sorry, thought it was clear...

```python
from pybricks.hubs import EssentialHub
from pybricks.tools import wait
from pybricks.parameters import Axis

# Initialize the hub.
hub = EssentialHub()

# Get the acceleration or angular_velocity along a single axis.
# If you need only one value, this is more memory efficient.
while True:

    # Read the forward acceleration.
    forward_acceleration = hub.imu.acceleration(Axis.X)

    # Read the yaw rate.
    yaw_rate = hub.imu.angular_velocity(Axis.Z)

    # Print the yaw rate.
    print(yaw_rate)
    wait(100)
```

**Reply by laurensvalk** [reply]
This feature was added only recently. Did you install the latest firmware on your hub? If you click on the hub at the bottom right, it should say something like shown above.

**Reply by ZPhilo** [reply]
No, I don't have the 3.3.0b4, only release version 3.2.3. Since these functions were already in documentation I thought they were already available in released version. Trying beta...

**Reply by laurensvalk** [reply]
Sounds good! If I understand correctly, this feature should only be shown in the documentation that matches the beta version.

To be sure, you can check the documentation in the app, which always matches.
